trust-quorum/gfss/src/gf256.rs

```diff
@@ -108,9 +108,9 @@ impl Binary for Gf256 {
     }
 }
 
-impl Distribution<Gf256> for Standard {
+impl Distribution<Gf256> for StandardUniform {
     fn sample<R: Rng + ?Sized>(&self, rng: &mut R) -> Gf256 {
-        Gf256(rng.r#gen())
+        Gf256(rng.random())
```

Comment:
Oh hell yes. I wanted the latest primarily because of this and the`StandardUniform` name change but was too lazy at the time to update it! Thanks @sunshowers!!!